packages/pluginsdk-peerdeps/.gitignore

```diff
@@ -0,0 +1 @@
+.scaffolddir
```

Comment:
The temp directory created for the scaffolded plugin is written into a file `.scaffolddir`